src/util/Affiliate.js

```diff
@@ -0,0 +1,15 @@
+
+function getTCGLink(card) {
+    let tcgplayerLink = 'https://shop.tcgplayer.com/';
+    if (card.details.tcgplayer_id) {
+      tcgplayerLink += `product/productsearch?id=${card.details.tcgplayer_id}`;
+    } else {
+      tcgplayerLink += `productcatalog/product/show?ProductName=${card.details.name}`;
+    }
+    tcgplayerLink += '&partner=CubeCobra&utm_campaign=affiliate&utm_medium=CubeCobra&utm_source=CubeCobra';
+  
+     return tcgplayerLink;
+}
+
+  
+export default { getTCGLink };
```

Comment:
Can you change the indentation in this file to 2 spaces and remove the empty line at the start?